src/shacl2code/lang/golang.py

```diff
@@ -140,6 +140,14 @@ class GolangRender(BasicJinjaRender):
 
     def __init__(self, args):
         super().__init__(args, TEMPLATE_DIR / "golang.j2")
+        self.__render_args = {
+                "module": args.module,
+        }
+
+    @classmethod
+    def get_arguments(cls, parser):
+        super().get_arguments(parser)
+        parser.add_argument("--module", help="Go module name")
```

Comment:
You probably want `required=True` to make this option required, since the option is required to generate correct bindings